src/result_gen.py

```diff
@@ -0,0 +1,563 @@
+                threesixty_pcd_numpy = preprocessing.pcd_processing_gapnet(
+                    self.artifact_threesixty_pcd_paths)
+        # TODO check which file is used depth/pcd
+        elif 'depthmap' in model_id:
+            if self.dataformat == 'depth':
+                front_pcd_numpy = preprocessing.get_depthmaps(
+                    self.artifact_front_pcd_paths)
+                back_pcd_numpy = preprocessing.get_depthmaps(
+                    self.artifact_back_pcd_paths)
+                threesixty_pcd_numpy = preprocessing.get_depthmaps(
+                    self.artifact_threesixty_pcd_paths)
+            else:
+                front_pcd_numpy = preprocessing.pcd_to_depthmap(
+                    self.artifact_front_pcd_paths, self.calibration)
+                back_pcd_numpy = preprocessing.pcd_to_depthmap(
+                    self.artifact_back_pcd_paths, self.calibration)
+                threesixty_pcd_numpy = preprocessing.pcd_to_depthmap(
+                    self.artifact_threesixty_pcd_paths, self.calibration)
+        else:
+            input_shape = json_metadata[0][0]['input_shape']
+            if self.dataformat == 'depth':
+                front_pcd_numpy = preprocessing.depthmap_to_pcd(
+                    self.artifact_front_pcd_paths, self.calibration, 'pointnet', input_shape)
+                back_pcd_numpy = preprocessing.depthmap_to_pcd(
+                    self.artifact_back_pcd_paths, self.calibration, 'pointnet', input_shape)
+                threesixty_pcd_numpy = preprocessing.depthmap_to_pcd(
+                    self.artifact_threesixty_pcd_paths, self.calibration, 'pointnet', input_shape)
+            else:
+                front_pcd_numpy = preprocessing.pcd_to_ndarray(
+                    self.artifact_front_pcd_paths, input_shape)
+                back_pcd_numpy = preprocessing.pcd_to_ndarray(
+                    self.artifact_back_pcd_paths, input_shape)
+                threesixty_pcd_numpy = preprocessing.pcd_to_ndarray(
+                    self.artifact_threesixty_pcd_paths, input_shape)
+
+        self.artifact_front_pcd_numpy = front_pcd_numpy
+        self.artifact_back_pcd_numpy = back_pcd_numpy
+        self.artifact_threesixty_pcd_numpy = threesixty_pcd_numpy
+
+    def get_height_per_artifact(self, model_id, service):
+        '''
+        For a list of artifacts, Get the Height prediction for each artifact in a list
+        '''
+        self.front_predictions = inference.get_predictions_2(
+            self.artifact_front_pcd_numpy, model_id, service)
+        self.back_predictions = inference.get_predictions_2(
+            self.artifact_back_pcd_numpy, model_id, service)
+        self.threesixty_predictions = inference.get_predictions_2(
+            self.artifact_threesixty_pcd_numpy, model_id, service)
+
+    def get_weight_per_artifact(self):
+        '''
+        Get the Weight prediction per artifact
+        '''
+        pass
```

Comment:
Should we remove this as it is unused. Maybe check for more occurrences of unused code

Reply:
TBD later in next PR